src/create_disk.sh

```diff
@@ -55,7 +56,13 @@ mount "${disk}2" rootfs/boot/efi
 
 # init the ostree
 ostree admin init-fs rootfs
-ostree pull-local "$ostree" "$ref" --repo rootfs/ostree/repo
+remote_arg=
+deploy_ref="${ref}"
+if [ "${remote_name}" != NONE ]; then
```

Comment:
This is checking against a string "NONE". Are we expecting to pass that string in if it doesn't exist?

Reply:
Yup, see comment higher up:

```
# use NONE here instead of empty string because this has to survive through various string processing 
```